docs/sphinx/user_guide/tutorial/indexset_segments.rst

```diff
@@ -98,14 +100,14 @@ RAJA IndexSets
 ^^^^^^^^^^^^^^^^^^^^^
 
 The ``RAJA::TypedIndexSet`` template is a container that can hold an 
```

Comment:
Remove "an" - "container that can hold any number".

Reply:
Fixed